backend/de.metas.manufacturing.rest-api/src/main/java/de/metas/handlingunits/rest_api/HandlingUnitsRestController.java

```diff
@@ -345,7 +359,8 @@ private ResponseEntity<JsonGetSingleHUResponse> getByIdSupplier(@NonNull final S
 	private static class GetByIdRequest
 	{
 		@NonNull HuId huId;
-		@Nullable HUQRCode expectedQRCode;
+		@Nullable
+		HUQRCode expectedQRCode;
```

Comment:
where is this used?

Reply:
there is no change here, it's just a result of formatting the page (ctrl + alt + L)